codebase/codebase_test.go

```diff
@@ -130,6 +130,35 @@ func (test *TestCodebaseRepository) TestListCodebases() {
 	assert.Equal(test.T(), codebase1.URL, codebases[0].URL)
 }
 
+func (test *TestCodebaseRepository) TestExistsCodebase() {
+	t := test.T()
+	resource.Require(t, resource.Database)
+
+	t.Run("codebase exists", func(t *testing.T) {
```

Comment:
Yes, the same issue we had with other tests when adding parallel. They shouldn't be affected but they are in fact